sdk/go/http/outbound_internals.go

```diff
@@ -56,44 +57,43 @@ func send(req *http.Request) (*http.Response, error) {
 		ptr: C.CString(req.URL.String()),
 		len: C.ulong(len(req.URL.String())),
 	}
-	spinReq.headers = toSpinReqHeaders(req.Header)
-	spinReq.body, err = toSpinReqBody(req.Body)
+	spinReq.headers = toWasiHeaders(req.Header)
```

Comment:
nit: Not sure I follow the new `toWasi*` functions — why use WASI in their name?

Reply:
Prefixing them as `outbound` sounds more appropriate to me.